### DISCUSSION TITLE
VScode formatting settings

### DISCUSSION THREAD

**Discussion by espenfjo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#572] [Q&A]
Hi @visualapproach

What settings/profile are you using for VSCode/clang-formatter?

Whenever I make a code change VScode heavily changes most of the existing formatting, which makes it a bit challenging to create PRs.

**Comment by visualapproach** [comment]
I don't use autoformatter. At least not that I'm aware of. Just did some change in the settings to default to the indent spacing.
